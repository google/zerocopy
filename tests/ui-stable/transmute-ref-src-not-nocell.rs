../ui-nightly/transmute-ref-src-not-nocell.rs