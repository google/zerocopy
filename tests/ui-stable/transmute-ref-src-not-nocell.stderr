error[E0277]: the trait bound `Src: NoCell` is not satisfied
  --> tests/ui-stable/transmute-ref-src-not-nocell.rs:22:32
   |
22 | const SRC_NOT_NO_CELL: &AU16 = transmute_ref!(&Src(AU16(0)));
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |                                |
   |                                the trait `NoCell` is not implemented for `Src`
   |                                required by a bound introduced by this call
   |
   = help: the following other types implement trait `NoCell`:
             bool
             char
             isize
             i8
             i16
             i32
             i64
             i128
           and $N others
note: required by a bound in `AssertSrcIsNoCell`
  --> tests/ui-stable/transmute-ref-src-not-nocell.rs:22:32
   |
22 | const SRC_NOT_NO_CELL: &AU16 = transmute_ref!(&Src(AU16(0)));
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `AssertSrcIsNoCell`
   = note: this error originates in the macro `transmute_ref` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `Src: NoCell` is not satisfied
  --> tests/ui-stable/transmute-ref-src-not-nocell.rs:22:32
   |
22 | const SRC_NOT_NO_CELL: &AU16 = transmute_ref!(&Src(AU16(0)));
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `NoCell` is not implemented for `Src`
   |
   = help: the following other types implement trait `NoCell`:
             bool
             char
             isize
             i8
             i16
             i32
             i64
             i128
           and $N others
note: required by a bound in `AssertSrcIsNoCell`
  --> tests/ui-stable/transmute-ref-src-not-nocell.rs:22:32
   |
22 | const SRC_NOT_NO_CELL: &AU16 = transmute_ref!(&Src(AU16(0)));
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `AssertSrcIsNoCell`
   = note: this error originates in the macro `transmute_ref` (in Nightly builds, run with -Z macro-backtrace for more info)
